android/app/src/main/java/labs/pooh/eaterslab/MapSearchActivity.kt

```diff
@@ -44,9 +50,18 @@ class MapSearchActivity : AbstractRevealedActivity() {
 
     private lateinit var gpsProvider: GpsMyLocationProvider
 
+    private lateinit var API: CafeteriaApi
+
+    /**
+     * Resources that should be disposed when the fragment is destroyed
+     */
+    private val compositeDisposable = CompositeDisposable()
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        API = CafeteriaApi(getString(R.string.api_url))
```

Comment:
I think API url should come from gradle, not from static strings. This way you can build debug with let's say `localhost` and target release builds against official API endpoint. To discuss.

Reply:
moved to gradle.properties. for now there is common version for debug/release but canbe easily changed